client/components/plans/plan-actions/index.jsx

```diff
@@ -155,12 +162,26 @@ module.exports = React.createClass( {
 			</div>;
 		}
 
+		var classes = classNames( {
```

Comment:
True. it used to have a condition and I thought I'd leave it there when I removed the conditionality. But yeah it's probably better as one line.